src/client/Microsoft.Identity.Client/AuthenticationResult.cs

```diff
@@ -315,6 +319,11 @@ internal AuthenticationResult() { }
         /// </summary>
         public AuthenticationResultMetadata AuthenticationResultMetadata { get; }
 
+        /// <summary>
+        /// Exposes the mTLS certificate used for authentication, if applicable.
+        /// </summary>
+        public X509Certificate2 MtlsCertificate { get; internal set; }
```

Comment:
This is passed in the request, right? Why pass back in response?

Reply:
+1. It will be needed for MSI integration, but not now.